### PR TITLE
Move msr functions into a package

### PR DESCRIPTION
**Pull request by GanShun**
Signed-off-by: Gan Shun Lim <ganshun@gmail.com>

### CODE REVIEW

File: cmds/core/msr/msr.go [hunk 1]
```diff
@@ -110,7 +111,7 @@ func rd(f forth.Forth) {
 	r := f.Pop().(uint32)
 	m := f.Pop().([]string)
 	forth.Debug("rd: %v %v", m, r)
-	data, errs := rdmsr(m, r)
+	data, errs := msr.ReadMSR(m, r)
```
Comment (rminnich) [inline cmds/core/msr/msr.go]: stutters, just make it msr.Read

File: cmds/core/msr/msr.go [hunk 2]
```diff
@@ -132,7 +133,7 @@ func wr(f forth.Forth) {
 	r := f.Pop().(uint32)
 	m := f.Pop().([]string)
 	forth.Debug("wr: %v %v %v", m, r, v)
-	errs := wrmsr(m, r, v)
+	errs := msr.WriteMSR(m, r, v)
```
Comment (rminnich) [inline cmds/core/msr/msr.go]: msr.Write

Comment (GanShun) [reply cmds/core/msr/msr.go]: Done

File: cmds/core/msr/msr.go (hunk 1 above)
Comment (GanShun) [reply cmds/core/msr/msr.go]: Done

**Pull request merged**